packages/console/package.json

```diff
@@ -90,20 +90,28 @@
     "rules": {
       "complexity": "off",
       "@typescript-eslint/prefer-nullish-coalescing": "off"
-    }
+    },
+    "overrides": [
+      {
+        "files": [
+          "*.test.ts"
+        ],
+        "rules": {
+          "@typescript-eslint/no-unsafe-return": "off"
```

Comment:
Do we have unsafe returns in our ac code?

Reply:
a lot in test files

Reply:
Hmmm, but we don't have test files in console, do we?

Reply:
ah, this is a mistake